src/pages/sns/faq.tsx

```diff
@@ -689,6 +698,64 @@ function SnsFaqPage() {
                 </Link>
               </ul>
             </Faq>
+          </FaqSection>
+
+          <FaqSection
+            id="treasury"
+            title={
+              <h2 className="tw-heading-3 text-gradient mb-12 md:mb-0 md:tw-heading-60">
+                SNS DAO treasury
+              </h2>
+            }
+          >
+            <Faq title="What is the SNS treasury?">
+              After a decentralization swap successfully concludes, the ICP acquired through the
+              swap is allocated to the SNS DAO's treasury. Optionally, the DAO can allocate a number of
+              SNS tokens to be in the DAO's treasury, which all teams so far have done. The SNS treasury
+              is controlled by the DAO, and can only be transferred using proposals that SNS token holders
+              vote on.
+            </Faq>
+
+            <Faq title="What is the purpose of the SNS treasury and how are the funds distributed?">
+              <ul>
+                <li>
+                  The purpose of the treasury is decided by the SNS DAO.
+                </li>
+                <li>
+                  A portion of the treasury can be sent to any address by an SNS proposal.
+                </li>
+                <li>
+                  Typically, after an SNS has launched it has an SNS token treasury and an ICP
+                  treasury (from the decentralization swap). An SNS can, however, also own other
+                  tokens on the Internet Computer.
+                </li>
+                <li>
+                  The treasury funds could, for example, be used to open liquidity pools on
+                  DEXs or to reward users for certain actions in the dapp. 
+                </li>
+              </ul>
+            </Faq>
+
+            <Faq title="Do I have a say in how the treasury funds are allocated?">
+              Yes, if you are a neuron holder of the respective SNS DAO, you
+              can actively exercise your voting power to decide on proposals that
+              allocate treasury funds.
+            </Faq>
+
+            <Faq title="Who has access to the treasury? ">
+              Without an adopted SNS proposal that specifies how to allocate funds,
+              no one can access the SNS treasury. The SNS DAO only grants access
+              when proposals are adopted through SNS community voting. It can happen
+              that the original developer team possesses a large portion of the voting
```

Comment:
I wonder if this can be perceived negatively. We could also say that this is a risk if any (centralised) party has a large portion of the voting power